radix-engine/tests/authorization_resource.rs

```diff
@@ -92,6 +97,29 @@ fn test_resource_auth(action: Action, update_auth: bool, use_other_auth: bool, e
                 "deposit_batch",
                 args!(Expression::entire_worktop()),
             ),
+        Action::Recall => {
+            let vaults = test_runner.get_component_vaults(account, token_address);
+            let vault_id = vaults[0];
+
+            builder
+                .add_instruction(Instruction::CallNativeMethod {
+                    method_ident: NativeMethodIdent {
+                        receiver: RENodeId::Vault(vault_id),
+                        method_name: "take".to_string(),
```

Comment:
Why is action `recall` and method `take`?

Reply:
`recall` action refers to "Direct Access" + "Take"